src/serializers/review.ts

```diff
@@ -0,0 +1,18 @@
+import { Review } from '@prisma/client'
+
+export function serializeReview(review: Review | null) {
+  if (review) {
+    return {
+      id: review.id,
+      type: 'review',
+      attributes: {
+        shelterId: review.shelterId,
+        cleanliness: review.cleanliness,
+        safety: review.safety,
+        staff: review.staff,
+      },
+    }
+  } else {
+    return { data: {} }
```

Comment:
This implementation was a good placeholder for 'errors' but I think now we are in a position to return real 404 errors and validation errors instead of simply an empty object. See #20 and #21

Reply:
If you would like to make a separate issue and PR for handling errors that is fine with me.

Reply:
I'm going to work on error handling separately.